handlers/get.go

```diff
@@ -58,13 +58,18 @@ func (d *Dependency) Get(c *fiber.Ctx) error {
 	// we need to replace escaped newline to literal newline
 	content := strings.Replace(i.Paste, `\n`, "\n", -1)
 
+	c.Set("Content-Type", "text/plain")
```

Comment:
the `Content-Type` headers will be set twice when `qs.Language` is not empty if you put it here

Reply:
alasan kenapa ditaruh disini relate sama ini https://github.com/teknologi-umum/polarite/pull/1#discussion_r722002352